modules/lib/src/main/resources/assets/js/app/settings/wizard/panel/ProjectWizardPanel.ts

```diff
@@ -108,6 +108,12 @@ export class ProjectWizardPanel
         }
     }
 
+    giveInitialFocus(): void {
+       this.whenRendered(() => {
```

Comment:
Maybe we should move this into the [parent class](https://github.com/enonic/lib-admin-ui/blob/master/src/main/resources/assets/admin/common/js/app/wizard/WizardPanel.ts#L706), so that the display name input gets focus after render in all of the wizards? Makes total sense to me.